samples/js/photos/src/types.js.flow

```diff
@@ -0,0 +1,47 @@
+// @flow
```

Comment:
As discussed: it needs to be in a .js.flow file because just .js is being picked up by eslint, which doesn't like the `declare` keyword.

... except it turns out it just doesn't like `declare export`, so a workaround is just not exporting them inline and instead have an `export {Face...}` at the end.